### PR TITLE
Lapr/updating project

### PR DESCRIPTION
**Pull request by LProulx86**
GitHub Issue: #

## Proposed Changes

 Prep for creating NuGet

## What is the current behavior?
Init of last map module.

## Checklist

Please check if your PR fulfills the following requirements:

- [x] Documentation has been added/updated (partial)
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: Nventive.Location.sln [hunk 1]
```diff
@@ -3,17 +3,17 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.1.32414.318
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Cartography", "Cartography", "{6643A3E0-7CD4-4BCE-84BD-0B24B1720DC2}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Nventive.Location", "Nventive.Location", "{6643A3E0-7CD4-4BCE-84BD-0B24B1720DC2}"
```
Comment (carlh98) [inline Nventive.Location.sln:6]: It should stay Cartography, we're not using Nventive.Location futhermore

File: README.md [hunk 2]
```diff
@@ -20,22 +20,23 @@ The following is the template for the final README.md file:
 
 ---
 
-# Project Title
+# Nventive.Location
```
Comment (carlh98) [inline README.md:23]: Same

File: src/Nventive.Location/Nventive.Location.DynamicMap/Nventive.Location.DynamicMap.csproj [hunk 3]
```diff
@@ -27,6 +30,10 @@
 		<PackageReference Include="Xamarin.Google.iOS.Maps" Version="3.9.0" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <ProjectReference Include="..\Nventive.Location.Core\Nventive.Location.Core.csproj" />
```
Comment (carlh98) [inline src/Nventive.Location/Nventive.Location.DynamicMap/Nventive.Location.DynamicMap.csproj:34]: same

**Review by carlh98 — CHANGES_REQUESTED** [review]

File: doc/MapService(umbrellaVersion).md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+| Shows directions by specifying LocationName              |  -  |    X    |  X  |
+| Show and center a specified location by LocationName     |  -  |    X    |  X  |
+
+## Usage
+
+### 1. Configure your application.
+
+- For all platforms
+
+    1. Add a reference to the Umbrella Map Service (Umbrella.Location.MapService)
```
[the remark this answers is not in the document]
Comment (LProulx86) [reply doc/MapService(umbrellaVersion).md:27]: Yeap TODO that's why it's umbrellaVersion. (file name)

**Review by carlh98 — APPROVED** [review]

**Pull request merged**